tower-hedge/src/latency.rs

```diff
@@ -42,15 +49,15 @@ where
 impl<S, R, Request> Service<Request> for Latency<R, S>
 where
     S: Service<Request>,
-    S::Error: Into<super::Error>,
+    super::Error: From<S::Error>,
```

Comment:
why this change?

Reply:
Because for some reason, if I keep it as `Into<super::Error>`, I get a tonne of errors saying that the error does not implement `Send` and `Sync`. I'm not entirely sure why inverting the bound fixes the issue?

Reply:
```
note: required because of the requirements on the impl of `std::convert::From<E>` for `std::boxed::Box<dyn std::error::Error + std::marker::Send + std::marker::Sync>`
```

Reply:
Actually, maybe @seanmonstar has some insight to provide here?

Reply:
Can we just use `Into<Box<dyn Error>>` or whatever?

Reply:
That's what `super::Error` is:
https://github.com/tower-rs/tower/blob/e32a5ecd0cd24f5fabd139310ba854a8583ca2c4/tower-hedge/src/lib.rs#L50

Reply:
You're seeing the error down in `Future::poll` because of the `?` operator, which uses `From` instead of `Into`.

Reply:
Thanks!